src/Modules/BuildList.lua

```diff
@@ -60,10 +60,25 @@ function listMode:Init(selBuildName, subPath)
 		self:SortList()
 	end)
 	self.controls.sort:SelByValue(main.buildSortMode, "sortMode")
-	self.controls.buildList = new("BuildListControl", {"TOP",self.anchor,"TOP"}, 0, 50, 640, 0, self)
+	-- self.controls.buildList = new("BuildListControl", {"TOP",self.anchor,"TOP"}, 0, 50, 640, 0, self)
+	-- self.controls.buildList.height = function()
+	-- 	return main.screenH - 58
+	-- end
+	self.controls.buildList = new("BuildListControl", {"TOP",self.anchor,"TOP"}, 0, 75, 640, 0, self)
 	self.controls.buildList.height = function()
-		return main.screenH - 58
+		return main.screenH - 80
 	end
+	self.controls.searchText = new("EditControl", {"TOP",self.anchor,"TOP"}, -29, 25, 340, 20, self.filterBuildList)
+	self.controls.searchTextLabel = new("LabelControl", {"RIGHT",self.controls.searchText,"LEFT"}, -4, 0, 0, 16, "^7Search for a Build:")
```

Comment:
This label is unnecessary, there is a parameter on EditControl to insert a label already.  See the search box on the TreeTab, for example.

Reply:
Resolved, remove label and changed EditControl in order to get default parameter or placeholder